frontend/app/components/redesign/components/revshare/ShareInput.tsx

```diff
@@ -11,34 +11,83 @@ interface ShareInputProps {
   weight: number
   percent: number
   placeholder?: string
-  percentDisabled?: boolean
   weightDisabled?: boolean
   onChangeName: (name: string) => void
   onChangePointer: (pointer: string) => void
   onChangeWeight: (weight: number) => void
-  onChangePercent: (percent: number) => void
   onRemove: () => void
   validatePointer: (pointer: string) => boolean
 }
 
 const GRID_COLS = 'md:grid-cols-[16rem_1fr_6rem_6rem_auto]'
 const GRID_GAP = 'md:gap-x-md'
 
+export const ShareInputTable = ({
+  children
+}: {
+  children: React.ReactNode
+}) => {
+  return (
+    <div
+      role="table"
+      aria-labelledby="revshare-table-caption"
+      className="contents"
+    >
+      <div id="revshare-table-caption" className="sr-only">
+        Revenue sharing recipients table
+      </div>
+      {children}
+    </div>
+  )
+}
+
 export const ShareInputHeader = () => {
```

Comment:
The header should be inside a `role="rowgroup"` (acts as `<thead>`), and then all the data inside another `role="rowgroup"` (as it already is, acts as `<tbody>`).